ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/management-security.yml

```diff
@@ -18,3 +18,13 @@ management_security:
       tls_versions: "1.1"
     - name: tls-single-version-profile-as-float
       tls_versions: 1.0
+    - name: test-trust-cert
+      trust_certificates:
+        - test-trust.crt
+        - requirement
+        - system
+    - name: test-chain-cert
+      chain_certificates:
+        - test-chain-cert1.crt
+        - test-chain-cert2.crt
+        - requirement include root-ca
```

Comment:
This seems like a hack. Are these proper CLI options? If so we should write the proper data model for them. Same for requirement & system above

Reply:
+1 with @ClausHolbechArista we should separate the list of certificate from the `requirement` ... options.

Given that `requirement basic-constraint ca true` can be in both chain and trust I think it would make more sense to use a `$defs` for this one